static/app/routes.tsx

```diff
@@ -1049,22 +1028,38 @@ function buildRoutes() {
         path=":projectId/events/:eventId/"
         component={errorHandler(ProjectEventRedirect)}
       />
+      <Route
+        path=":projectId/getting-started/"
+        component={make(
+          () => import('sentry/views/projectInstall/platformOrIntegration')
+        )}
+      />
     </Fragment>
   );
   const projectsRoutes = (
     <Fragment>
       {usingCustomerDomain && (
         <Route
           path="/projects/"
-          component={withDomainRequired(NoOp)}
+          component={make(async () => {
+            const {Projects} = await import('sentry/views/projects/');
+            return {
+              default: Projects,
+            };
```

Comment:
nit, don't use non-default exports for views right now. We don't have a precedent for this.

(At some point in the future I may look at supporting this, but likely what will happen is that it pulls out attributes that look like `*View`)

Reply:
I already introduced this in some of my previous PRs... I can remove this introduction in the other routes too in a separate PR, but it would be really nice to have named exports everywhere

Reply:
It will, but I think for views this is just extraneous boiler plate at the moment since there no large advantage to named exports for views right now.

once we’re closer to having named exports everywhere we can come up with a better way to have the import extracted in a type safe way without the extra boiler plate like we have here